filter_plugins/insert_kernel_modules.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+import cpuinfo
+
+
+def get_cpu_info():
+    info = cpuinfo.get_cpu_info()
```

Comment:
2 points:
- ansible filters are for data manipulation. They are not intended to execute any actions / data collection on hosts
- code of filters are executed centrally on ansible controller, not on target hosts. Hence this will not work

cpu information will be needed to collecto on role level, like:
handle_module: "{{ cpu_constraints | check_cpu_constraints(cpu_family, cpu_model) }}"

Reply:
Thank you for the feedback. Should be addressed in 33036e3